hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -353,6 +353,8 @@ private static EthTxData populateLegacyEthTxData(RLPItem rlpItem, byte[] rawTx)
         byte recId = vBI.testBit(0) ? (byte) 0 : 1;
         if (vBI.compareTo(BigInteger.valueOf(34)) > 0) {
```

Comment:
What happens if vBI is outside of the ranges tested?  Should we throw an exception or would something catch this case down the line?

Reply:
when we are in the case of `LEGACY_ETHEREUM` txs we go through [one of these two cases](https://github.com/hashgraph/hedera-services/blob/11401-eip155-support/hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxSigs.java#L71-L80) where we set the `v` value